{"status": "broken", "attachments": [{"source": "5a2834ef-19c5-49d9-b853-c8cb85ed08d3-attachment.txt", "type": "text/plain", "name": "log"}, {"source": "d1ddd3dc-4dea-4655-b312-3296001519ca-attachment.txt", "type": "text/plain", "name": "stdout"}, {"source": "1ca9738a-a1d1-4419-b8d8-af71d9f730b7-attachment.txt", "type": "text/plain", "name": "stderr"}], "labels": [{"name": "suite", "value": "test_case_api"}, {"name": "host", "value": "DESKTOP-CLD22EL"}, {"name": "thread", "value": "18228-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython2"}, {"name": "package", "value": "test_case_api"}], "stop": 1560331397327, "uuid": "d8415aea-abcf-4f6d-9c5f-6b4062eb5c4f", "historyId": "99f9dab82bcd4d36e8febe9983b203f5", "start": 1560331397264, "fullName": "test_case_api#test_businessmen_can_use_channel", "statusDetails": {"message": "UnicodeEncodeError: 'ascii' codec can't encode characters in position 552-555: ordinal not in range(128)", "trace": "def test_businessmen_can_use_channel():\n        print(\"----------------------------商户可用通道查询--------------------------\")\n        paths = \"/channel/showMerchantChannelType.htm\"\n        pam = {\"partnerId\": global_parameter.PARTNERID, \"merNo\": global_parameter.MERNO_NO_USERID}\n    \n        r_obj = Public_Request()\n        r_resu = r_obj.public_request(pas=paths, parameters=pam)\n    \n>       print(json.dumps(r_resu.json(), indent=2, sort_keys=False, ensure_ascii=False))\nE       UnicodeEncodeError: 'ascii' codec can't encode characters in position 552-555: ordinal not in range(128)\n\ntest_case_api.py:46: UnicodeEncodeError"}, "name": "test_businessmen_can_use_channel"}